jenkins_job_wrecker/modules/triggers.py

```diff
@@ -271,26 +282,39 @@ def ghprbtrigger(top, parent):
         elif child.tag == 'autoCloseFailedPullRequests':
             ghpr['auto-close-on-fail'] = get_bool(child.text)
         elif child.tag == 'blackListCommitAuthor':
-            ghpr['black-list-commit-author'] = child.text.strip().split(' ')
+            if child.text:
+                ghpr['black-list-commit-author'] = child.text.strip().split(' ')
+            else:
+                ghpr['black-list-commit-author'] = []
         elif child.tag == 'blackListLabels':
-            ghpr['black-list-labels'] = child.text.strip().split('\n')
+            if child.text:
+                ghpr['black-list-labels'] = child.text.strip().split('\n')
+            else:
+                ghpr['black-list-labels'] = []
         elif child.tag == 'blackListTargetBranches':
-            ghpr['black-list-target-branches'] = [item[0].text for item in child]
+            black_list_target_branches = [item[0].text.strip() for item in child if item[0].text is not None]
```

Comment:
`black_list_target_branches` seems unnecessary, especially that the nearly identical code for `whiteListTargetBranches` tag doesn't introduce a temporary variable